docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -182,13 +182,28 @@ Add the following property to `hive-site.xml` in your Hive installation `conf` d
 </property>
 ```
 
-To use fault tolerant mode, set Alluxio scheme to be `alluxio-ft`:
+To use fault tolerant mode, set the Alluxio cluster properties appropriately in an
```

Comment:
what is "appropriately"? I think simply setting `conf/alluxio-site.properties` in Alluxio will not work with Hive, right?